services/movie.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None

```

Comment:
These parameters are Optional[List[int]] because they accept None as default
Fix in all places

Reply:
Done everywhere